enhancements/monitoring/hypershift-monitoring.md

```diff
@@ -0,0 +1,392 @@
+* We optimized new hires for the Observability Platform team for skillsets to solve those problems. We are hiring for the 2-3 slots.
+
+Mitigations:
+
+* We are growing the dev team even more.
+* We are doubling down the development on performance and scalability. We already have designs to improve the biggest bottlenecks of Thanos.
+* In the extremely worst case: This design assumes API-driven workflows. This allows easy change of Observability Product if anything goes wrong. If our Observability Product effort totally collapses, OSD can within days (and millions $$) switch to any other vendors that support Loki APIs and Remote Write API and Prometheus/Loki read APIs. For example:
+    * Our Grafana Cloud partner
+    * Amazon Managed Prometheus and Grafana
+    * Google Managed Prometheus
+    * Logz.io
+    * … many more
+
+### Migration
+
+N/A. HyperShift is a new project.
+
+### Documentation Required
+
+User facing docs:
+
+* HyperShift client monitoring and how to configure metrics remote write (push API) to desired backend.
+
+Internal docs:
+
+* RHOBS entry-points and usage.
+
+### Open Questions
+
+### FAQ
+
+Some common questions gathered during initial proposal discussions.
+
+#### What About Layered Service / Addons Monitoring?
+
+This was discussed and solved by [Monitoring Stacks](monitoring-stack-operator.md). For HyperShift context, Addons requirements are no different to what we assume Customer would want to do / use. So in the context of this enhancement Addons are just another CUS.
+
+TBD explain risk in terms of having Addons monitoring running on "unmanaged" data-plane.
+
+#### How HyperShift Admin is any different from a Cluster Administrator?
+
+They manage multiple clusters in HyperShift topology, so multiple control planes on single management OSD. So there is added complexity on top. But technically they are administrating clusters. 
+
+#### How this can work for on-premise HyperShift?
+
+As mentioned in [Big picture](#the-big-picture), for CUS local use, metrics can be shipped to many different solutions. For example:
+
+* Data can land to locally deployed Observatorium. Obsevatorium is our open-source project that can already be installed anywhere to fulfil the proposed architecture. The first time user onboarding experience is certainly less than optimal, though a decent level of investment in better documentation, reusable dashboards and alerts and the operator deployment model would make this a drastically better experience.
+* 3rd-party vendor like Grafana Cloud, Amazon or Google Managed Prometheus, Logz.io and others.
+
+Exact offering is still to be discussed, but we need to acknowledged that managing monitoring is and expensive business.
+
+### Test Plan
+
+TBD
+
+## Implementation History
+
+If all parties will agree on the proposed direction we need heavy focus on the actions required:
+
+1. Add Alerting on top of Observatorium, then deploy it on RHOBS.
```

Comment:
This is an implementation history of an obvervability product, not of monitoring in Hypershift.